clr.coreclr.props

```diff
@@ -78,7 +78,7 @@
     <!-- The rejit feature is available only on supported architectures (x86 & x64) -->
     <FeatureReJIT Condition="('$(TargetArch)' == 'i386') or ('$(TargetArch)' == 'amd64')">true</FeatureReJIT>
     <FeatureWindowsPhone>true</FeatureWindowsPhone>
-    <FeatureManagedEtw>true</FeatureManagedEtw>
+    <FeatureManagedEtw Condition="'$(OS)' != 'Unix'">true</FeatureManagedEtw>
```

Comment:
Ok, I'll change this as you suggest. If the OS will not be Unix, though, we'll have many places to update, besides this one.